init-tools.cmd

```diff
@@ -0,0 +1,72 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+set INIT_TOOLS_LOG=%~dp0init-tools.log
+if [%PACKAGES_DIR%]==[] set PACKAGES_DIR=%~dp0.nuget\
+if [%TOOLRUNTIME_DIR%]==[] set TOOLRUNTIME_DIR=%~dp0build_tools
+set DOTNET_PATH=%~dp0.dotnet_stage0\
+if [%DOTNET_CMD%]==[] set DOTNET_CMD=%DOTNET_PATH%dotnet.exe
+if [%BUILDTOOLS_SOURCE%]==[] set BUILDTOOLS_SOURCE=https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json
+set /P BUILDTOOLS_VERSION=< "%~dp0BuildToolsVersion.txt"
+set BUILD_TOOLS_PATH=%PACKAGES_DIR%Microsoft.DotNet.BuildTools\%BUILDTOOLS_VERSION%\lib\
+set PROJECT_JSON_PATH=%TOOLRUNTIME_DIR%\%BUILDTOOLS_VERSION%
+set PROJECT_JSON_FILE=%PROJECT_JSON_PATH%\project.json
+set PROJECT_JSON_CONTENTS={ "dependencies": { "Microsoft.DotNet.BuildTools": "%BUILDTOOLS_VERSION%" }, "frameworks": { "netcoreapp1.0": { } } }
+set BUILD_TOOLS_SEMAPHORE=%PROJECT_JSON_PATH%\init-tools.completed
+set DOTNET_INSTALL_ROOT_DIR=%~dp0.dotnet_stage0
+
+if DEFINED "%CHANNEL%" (
```

Comment:
Ugh the drawbacks of switching between OSX and Windows...one of my changes was dropped that defined this. Will add it back